Code/Framework/AzQtComponents/AzQtComponents/Components/FancyDocking.cpp

```diff
@@ -286,6 +285,9 @@ namespace AzQtComponents
         m_perScreenFullScreenWidgets.clear();
 #endif
 
+#if !defined(AZ_PLATFORM_WINDOWS)
+        QRect totalScreenRect;
+#endif
```

Comment:
This seems strange that the totalScreenRect is only calculated on Windows and used to set the geometry.
@cgalvan Do you know why this is the case.
Also we should have some comments explaining why Windows has a different behavior